packages/web-components/src/components/ic-menu/ic-menu.tsx

```diff
@@ -603,7 +603,10 @@ export class Menu {
         if (this.isSearchBar) {
```

Comment:
Is it worth moving this code into a separate function to avoid duplication? Seems to be exactly the same logic as in the default case when (event.key !== "Tab")